.github/workflows/test.yml

```diff
@@ -0,0 +1,103 @@
+name: tests
+on: [pull_request]
+jobs:
+
+  unit-tests:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: 2.4.6
+        bundler-cache: true
+    - uses: ./.github/actions/setup-test-environment
```

Comment:
Aha - ok so this is missing this file, which we made in plots2 to reduce the length of the script: https://github.com/publiclab/plots2/blob/main/.github/actions/setup-test-environment/action.yml

Reply:
aha! thanks Jeff